components/Wrap.vue

```diff
@@ -142,15 +151,17 @@ export default {
     async unwrap() {
       const signer = await this.provider.getSigner();
       const signerAddr = await signer.getAddress();
-      if (signerAddr.toLowerCase() != this.ad.owner) {
```

Comment:
Removed this because it's not a super helpful check and also I'm afraid of relying on our ad.owner override for when we switch to initState which doesn't have it.